### PR TITLE
Add justify-items CSS property

### LINKED ISSUE #149: css/properties.json is incomplete
Some properties that are missing from css/properties.json:

* justify-items
* justify-self

### PR DESCRIPTION
**Pull request by pkuczynski**
Based on:

https://drafts.csswg.org/css-align-3/#propdef-justify-items
https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

Fixes #149, https://github.com/csstree/csstree/issues/81

### CODE REVIEW

**Comment by lahmatiy** [comment]
Looks like `CSS Box Alignment Module` is a new group. It should be renamed to `CSS Box Alignment` and added to `css/definitions.json`

**Comment by pkuczynski** [comment]
Sure! Done...

File: css/properties.json [hunk 1]
```diff
@@ -4667,6 +4667,21 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "justify-items": {
+    "syntax": "normal | stretch | <baseline-position> | [ <overflow-position>? <self-position> ] | [ legacy || [ left | right | center ] ]",
```
Comment (wbamberg) [inline css/properties.json]: It looks as if this has changed: https://drafts.csswg.org/css-align-3/#propdef-justify-items. If you agree, and can update it, I'll merge this PR.

**Comment by pkuczynski** [comment]
@wbamberg I updated the PR. Is it good now? It's annoying that it's stuck for so long. I was planning to add more stuff, but my PR has not been merged so my motivation floored.

**Comment by wbamberg** [comment]
>  It's annoying that it's stuck for so long.

I know, I'm sorry about that. Thanks for your contribution. It looks good to me now, and since it already got the 👍 from lahmatiy, let's merge this.

**Comment by pkuczynski** [comment]
No worries, we all busy :)

**Comment by pkuczynski** [comment]
So now time for #184 :)

**Comment by frenic** [comment]
I'm thankful too @pkuczynski for all your PRs. I would love to see more from you 🙏

**Comment by pkuczynski** [comment]
We will see @frenic ;)

**Pull request merged**